vars/perfRegressionParallelPipeline.groovy

```diff
@@ -106,6 +106,10 @@ def call(Map pipelineParams) {
             string(defaultValue: '',
                    description: 'Actual user requesting job start, for automated job builds (e.g. through Argus)',
                    name: 'requested_by_user')
+            string(defaultValue: "${pipelineParams.get('perf_extra_jobs_to_compare', '')}",
```

Comment:
how about setting default to e.g. regression tests (better would be to official runs, but there's too many versions now and would made the report too long)?
This way, running from staging for the first time would produce the report (currently fails due missing reference) and would give also comparison to weekly results.

Reply:
that's exactly what I was planning todo
but first need to define their "official" locations